packages/flutter_tools/test/general.shard/args_test.dart

```diff
@@ -46,6 +46,32 @@ void main() {
     expect(command.boolArg(FlutterGlobalOptions.kContinuousIntegrationFlag, global: true), true);
   });
 
+  testUsingContext('Global arg results are available in FlutterCommands sub commands', () async {
+    final DummyFlutterCommand command = DummyFlutterCommand(
+        commandFunction: () async {
+          return const FlutterCommandResult(ExitStatus.success);
+        }
+    );
+
+    final DummyFlutterCommand subcommand = DummyFlutterCommand(
+      name: 'sub',
+        commandFunction: () async {
+          return const FlutterCommandResult(ExitStatus.success);
+        }
```

Comment:
```suggestion
      commandFunction: () async {
        return const FlutterCommandResult(ExitStatus.success);
      },
```

Reply:
I think you're looking at old version, that should already be fixed

Reply:
whoops